jupyterlab_bigquery/jupyterlab_bigquery/list_items_handler/service.py

```diff
@@ -87,6 +87,7 @@ def list_models(self, dataset_id):
         'id': model_full_id,
         'name': model.model_id,
         'datasetId': dataset_id,
+        'type': 'MODEL'
```

Comment:
Let's make this a const at the top near SCOPE:

```
MODEL = 'MODEL'
```

Reply:
Got it